vm/compiler/src/lib.rs

```diff
@@ -67,6 +67,29 @@ fn substitute_variable<S: ::std::hash::BuildHasher>(
     .to_string()
 }
 
+/// check a string contains any Windows style line ending (CRLF)
+/// and perform the conversion to Unix stype (LF) if yes
+fn windows_line_ending_to_unix(text: &str) -> String {
+    let mut converted = String::new();
+    for c in text.chars() {
```

Comment:
you can just replace "\r\n" with "\n", '\r' alone is still allowed I think.

Reply:
@zhutoulala  Can you help to resolve this and rebase this pr?

Reply:
Hello. `\r` is usually used with `\n` in most cases. It alone is still allowed, it means to return to the beginning of the current line though, so I doubt if anyone wants to do that in move. I can update this PR with a quick `.replace("\r\n", "\n")` but it could be a bit slower. Let me know if that's what you want.